ern-container-gen-ios/src/hull/ElectrodeContainer/DevAssist/ERNDevSettingsViewController.m

```diff
@@ -82,10 +82,7 @@ - (void)viewDidLoad {
     if ([[NSUserDefaults standardUserDefaults] objectForKey:packagerIPPort] == nil) {
         self->packagerIPandPort = @"localhost:8081";
         [[NSUserDefaults standardUserDefaults] setObject:self->packagerIPandPort forKey:packagerIPPort];
-    }
-    // Store it in NSUserDefaults. default it to YES
-//    [[NSUserDefaults standardUserDefaults] setBool:YES forKey:autoReloadBundle];
-    
+    }    
```

Comment:
Trailing spaces here. Please remove.

Reply:
👍

Reply:
Still there.. ;)


![image](https://user-images.githubusercontent.com/743291/66082711-8df68100-e51f-11e9-82bf-ec650c649a89.png)

Reply:
I'm wondering how you are able to see such dotted lines..

Reply:
Using this extension:
https://chrome.google.com/webstore/detail/render-whitespace-on-gith/ifdbipohclgnokjgpejhnbjdlgjkkhom?hl=en

Reply:
Sure 👍